app/assets/stylesheets/components/about.scss

```diff
@@ -134,13 +134,13 @@
   .contribute {
     color: #999;
```

Comment:
Can you define a variable for this color in `settings.scss` and then use it here?